bundles/net.tourbook.photo/src/net/tourbook/photo/ImageUtils.java

```diff
@@ -210,28 +245,12 @@ private static Image resize(final Display display, final Image image, final int
 
       Image scaledImage = new Image(display, imgWidth, imgHeight);
 
-      // For images with a transparent layer, this will keep the existing
-      // transparency.
-      // Source: https://stackoverflow.com/a/63703052
-      final ImageData origData = srcImage.getImageData();
-      if (origData.alphaData != null) {
+      final ImageData scaledImageWithTransparencyData =
+            copyImageTransparencyData(srcImage.getImageData(), imgWidth, imgHeight);
+      if (scaledImageWithTransparencyData != null) {
 
-         final ImageData destData = new ImageData(imgWidth, imgHeight, origData.depth, origData.palette);
-
-         destData.alphaData = new byte[destData.width * destData.height];
-         for (int destRow = 0; destRow < destData.height; destRow++) {
-
-            for (int destCol = 0; destCol < destData.width; destCol++) {
-
-               final int origRow = destRow * origData.height / destData.height;
-               final int origCol = destCol * origData.width / destData.width;
-               final int o = origRow * origData.width + origCol;
-               final int d = destRow * destData.width + destCol;
-               destData.alphaData[d] = origData.alphaData[o];
-            }
-         }
          UI.disposeResource(scaledImage);
-         scaledImage = new Image(display, destData);
+         scaledImage = new Image(display, scaledImageWithTransparencyData);
       }
 

```

Comment:
It might be possible that this photo is from me as I actually own those poles but out of precaution, I will change it with an image that we know for sure is not copyrighted

Reply:
Fixed and used a MyTourbook PNG image